### PR TITLE
Added time to Stammtisch event page and added page for next meeting

### PR DESCRIPTION
**Pull request by rxxx03**

### CODE REVIEW

File: _events/9.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title:  "Meeting #5 🌳"
```
Comment (shushugah) [inline _events/9.md:2]: remove the tree emoji, this was climate strike specific and seems out of place here, what do you think?

File: _events/8.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ title:  "Stammtisch #2 🍻"
 date:   2019-10-02 20:00:00 +0200
 ---
 
-## {{ page.date | date: '%A, %d %B %Y' }}
+## {{ page.date | date: '%A, %d %B %Y at %H:%M' }}
```
Comment (shushugah) [inline _events/8.md:6]: nice!

**Review by shushugah — APPROVED** [review]

File: _events/9.md (hunk 1 above)
Comment (rxxx03) [reply _events/9.md:2]: done.

Comment (rxxx03) [reply _events/9.md:2]: not sure how to add changed to a pull request that has been accepted, so i created a new pull request... :)

**Pull request merged**